ricecooker/classes/nodes.py

```diff
@@ -203,25 +208,25 @@ class ContentNode(Node):
         Base model for different content node kinds (topic, video, exercise, etc.)
 
         Attributes:
-            id (str): content's original id
+            source_id (str): content's original id
             title (str): content's title
             description (str): description of content (optional)
             author (str): who created the content (optional)
             license (str): content's license based on le_utils.constants.licenses (optional)
```

Comment:
Would be good to use a class-based approach for licenses, as well, for consistency and to support validation. `copyright_holder` could then be an argument to the `License` object, and be required for `CCByNCSALicense`, for example, but not be required for `PublicDomainLicense`.